index.html

```diff
@@ -8289,8 +8289,9 @@ <h3>Ticks</h3>
  dispatch steps</dfn> on a <a>browsing context</a> <var>context</var>,
  and a <var>list of events</var> and their properties. These steps
  must be equivalent to performing the given input device manipulations
- on <var>context</var>, such that trusted events corresponding to the
- entries in <var>list of events</var>are dispatched.
+ on the top-browsing context ancestor of the <var>context</var>, such
```

Comment:
isn't the term "top-level browsing context"? could you link to the getter steps here https://html.spec.whatwg.org/multipage/document-sequences.html#bc-tlbc?

Reply:
Thanks! Replaced both usages.